src/main/java/com/hivemq/cli/ioc/HiveMqModule.java

```diff
@@ -51,34 +59,52 @@ class HiveMqModule {
+            final @NotNull ListDataPoliciesCommand listDataPoliciesCommand,
+            final @NotNull CreateDataPolicyCommand createDataPolicyCommand,
+            final @NotNull DeleteDataPolicyCommand deleteDataPolicyCommand,
+            final @NotNull BehaviorPolicyCommand behaviorPolicyCommand,
+            final @NotNull GetBehaviorPolicyCommand getBehaviorPolicyCommand,
+            final @NotNull UpdateBehaviorPolicyCommand updateBehaviorPolicyCommand,
+            final @NotNull ListBehaviorPoliciesCommand listBehaviorPoliciesCommand,
+            final @NotNull CreateBehaviorPolicyCommand createBehaviorPolicyCommand,
+            final @NotNull DeleteBehaviorPolicyCommand deleteBehaviorPolicyCommand,
+            final @NotNull BehaviorStateCommand behaviorStateCommand,
+            final @NotNull GetBehaviorStateCommand getBehaviorStateCommand,
+            final @NotNull SchemaCommand schemaCommand,
             final @NotNull GetSchemaCommand getSchemaCommand,
-            final @NotNull ListSchemasCommand listSchemasCommand,
+            final @NotNull ListSchemaCommand listSchemaCommand,
             final @NotNull CreateSchemaCommand createSchemaCommand,
             final @NotNull DeleteSchemaCommand deleteSchemaCommand,
             final @NotNull CommandLineConfig config,
             final @NotNull CommandErrorMessageHandler handler) {
 
-        final CommandLine policiesCommandLine = new CommandLine(policiesCommand).addSubcommand(getPolicyCommand)
-                .addSubcommand(updatePolicyCommand)
-                .addSubcommand(listPoliciesCommand)
-                .addSubcommand(createPolicyCommand)
-                .addSubcommand(deletePolicyCommand);
-        final CommandLine schemasCommandLine = new CommandLine(schemasCommand).addSubcommand(getSchemaCommand)
-                .addSubcommand(listSchemasCommand)
+        final CommandLine dataPolicyCommandLine = new CommandLine(dataPolicyCommand).addSubcommand(getDataPolicyCommand)
+                .addSubcommand(updateDataPolicyCommand)
+                .addSubcommand(listDataPoliciesCommand)
+                .addSubcommand(createDataPolicyCommand)
+                .addSubcommand(deleteDataPolicyCommand);
+        final CommandLine behaviorPolicyCommandLine =
+                new CommandLine(behaviorPolicyCommand).addSubcommand(getBehaviorPolicyCommand)
+                        .addSubcommand(updateBehaviorPolicyCommand)
+                        .addSubcommand(listBehaviorPoliciesCommand)
+                        .addSubcommand(createBehaviorPolicyCommand)
+                        .addSubcommand(deleteBehaviorPolicyCommand);
+        final CommandLine behaviorStateCommandLine =
+                new CommandLine(behaviorStateCommand).addSubcommand(getBehaviorStateCommand);
+        final CommandLine schemaCommandLine = new CommandLine(schemaCommand).addSubcommand(getSchemaCommand)
+                .addSubcommand(listSchemaCommand)
```

Comment:
listDataPolicies, listBehaviorPolicies but listSchema instead of listSchemas? Hmm